sidecar/config/flags.go

```diff
@@ -211,6 +212,11 @@ var Flags = []cli.Flag{
 		EnvVar: envVar(logLevelFlag),
 		Usage:  "Logging level (debug, info, warn, error, fatal, panic)",
 	},
+	cli.IntFlag{
+		Name:   discoveryPortFlag,
+		EnvVar: envVar(discoveryPortFlag),
+		Usage:  "Service discovery server port number",
```

Comment:
Maybe we shouldn't expose the SDS port as a configuration option? The only thing that uses it is Envoy, so it seems like an implementation detail that people should not have to care about.